### PR TITLE
GH1603: Push Cake.NuGet to MyGet/NuGet

### PR DESCRIPTION
**Pull request by mholo65**
Solves #1603

### CODE REVIEW

File: src/Cake.NuGet/Cake.NuGet.csproj [hunk 1]
```diff
@@ -1,12 +1,21 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <Description>Provides aliases (extension methods on Cake context) that support CI, build, unit tests, zip, signing, etc. for Cake.</Description>
```
Comment (mholo65) [inline src/Cake.NuGet/Cake.NuGet.csproj]: This is wrong!

Comment (mholo65) [inline src/Cake.NuGet/Cake.NuGet.csproj]: Any suggestion for this? @cake-build/cake-team 

Old packages on MyGet seems to use same description https://www.myget.org/feed/cake/package/nuget/Cake.NuGet/0.12.0

Comment (devlead) [inline src/Cake.NuGet/Cake.NuGet.csproj]: It's a nuget module right? So correct should be a better description

Comment (devlead) [inline src/Cake.NuGet/Cake.NuGet.csproj]: Perhaps:
`Cake Module providing NuGet install capabilities for i.e. preprocessor directives like #addin, #tool & #load`
?

Comment (mholo65) [inline src/Cake.NuGet/Cake.NuGet.csproj]: Sounds good to me!

**Comment by devlead** [comment]
@mholo65  your changes have been merged, thanks for your contribution 👍

**Pull request merged**